plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
The below code was correctly formated...

```suggestion
```

Reply:
fixed